docs/proposals/multiple-schedulers.md

```diff
@@ -152,6 +152,11 @@ which the client has set a scheduler annotation that does not correspond to a ru
 2. Dynamic launching scheduler(s) and registering to admission controller (as an external call). This also
 requires some work on authorization and authentication to control what schedulers can write the /binding
 subresource of which pods.
+3. Optimize the behaviors of priority functions in multi-scheduler scenario. In the case where multiple schedulers have
+the same predicate and priority functions (for example, when using multiple schedulers for parallelism rather than to
+customize the scheduling policies), all schedulers would tend to pick the same node as "best". To solve this problem,
```

Comment:
Your are right. I will rewrite the statement according to your suggestion, making it less misleading.